src/Administration/Resources/app/administration/src/app/component/data-grid/sw-data-grid/index.js

```diff
@@ -299,6 +313,21 @@ Component.register('sw-data-grid', {
             this.findResizeColumns();
         },
 
+        initCompactModeAndShowPreviews() {
+            if (this.identifier) {
```

Comment:
You can invert this condition as well, can't you?

Reply:
Yep, seems better like this! Fixed.